commons-numbers-complex/src/main/java/org/apache/commons/numbers/complex/Complex.java

```diff
@@ -2464,7 +1650,7 @@ public Complex asinh() {
         // Note: This is the opposite to the identity defined in the C99 standard:
```

Comment:
This comment is not relevant any more. However it is missing from ComplexFunctions so you should move it there.